### PR TITLE
Update assignment1/softmax.ipynb

### PR DESCRIPTION
**Pull request by jiunbae**
loss function은 손실함수
nested loop는 중첩 루프로 번역했는데 더 나은 번역이 있다면 알려주세요.

### CODE REVIEW

**Comment by myungsub** [comment]
현재 상태도 좋은 것 같습니다. 수고하셨습니다!

**Comment by jiunbae** [comment]
아 이거 번역이 다 되지 않았던거 같아서 추가로 번역했습니다.
